custom_components/elevenlabs_tts/tts.py

```diff
@@ -50,7 +57,12 @@ def default_language(self) -> str:
     @property
     def supported_languages(self) -> list[str]:
         """Return list of supported languages."""
-        return ["en", "de", "pl", "es", "it", "fr", "pt", "hi"]
+        return ["en"]
```

Comment:
This was added in #6 to fix an issue with the service not being available for users with other localizations. Although I wasn't able to reproduce that issue when changing my user accounts language, is there any harm in listing the multilingual model's supported voices?

Reply:
re-introduced :)